apps/web/store/bulkBookmarksAction.tsx

```diff
@@ -0,0 +1,44 @@
+// reference article https://refine.dev/blog/zustand-react-state/#build-a-to-do-app-using-zustand
+import { create } from "zustand";
+
+import type { ZBookmark } from "@hoarder/shared/types/bookmarks";
+
+interface BookmarkState {
+  selectedBookmarks: ZBookmark[];
+  isBulkEditEnabled: boolean;
+  handleBulkEdit: (isEnabled: boolean) => void;
```

Comment:
This is a small nitpicking, but it wasn't clear to me what `handle` here means. Maybe `setIsBulkEditEnabled`?

Reply:
Renamed